doc/3_selection.md

```diff
@@ -76,7 +76,7 @@ partition which is created for the RDD.
 
 ### Grouping rows by partition key
 
-Physically, cassandra stores data already grouped by Cassandra partition key and ordered by clustering
+Physically, Cassandra stores data already grouped by partition key and ordered by clustering
 column(s) within each partition. As a single Cassandra partition never spans multiple Spark partitions,
 it is possible to very efficiently group data by partition key without shuffling data around.
 Call `spanBy` or `spanByKey` methods instead of `groupBy` or `groupByKey`:
```

Comment:
We meed to specify within this that this only works for sequentially ordered data. So span's must still follow clustering key order see SPARKC-127